master/buildbot/schedulers/forcesched.py

```diff
@@ -654,11 +654,13 @@ def force(self, owner, builderNames=None, **kwargs):
         reason = self.reason.getFromKwargs(kwargs)
         if owner is None:
             owner = self.username.getFromKwargs(kwargs)
+        user_id = kwargs.get('user_id')[0] if kwargs.get('user_id') else None
```

Comment:
minor: If `kwargs` is a dictionary, you can use `kwargs.get('field', None)`

Reply:
You right but consider a situation when `kwargs['user_id'] == []`. My `if` checks `None` and `[]`